apps/spruce/src/pages/version/TestAnalysis/index.tsx

```diff
@@ -0,0 +1,235 @@
+            across more than one task
+          </Title>
+          <Body>
+            This page provides an overview of all test failures in the current
+            version. If a test fails across multiple tasks, it could suggest a
+            flaky test or a broader issue.
+          </Body>
+          <FilterContainer>
+            <div>
+              <TextInputWithValidation
+                aria-labelledby="test-failure-search-label"
+                disabled={!hasResults}
+                id="test-failure-search-input"
+                label="Search Test Failures"
+                onChange={(value) => {
+                  setTestName(value);
+                }}
+                placeholder="Search failed tests (regex)"
+                validator={validateRegexp}
+              />
+            </div>
+            <Combobox
+              disabled={!hasResults}
+              label="Failure type"
+              multiselect
+              onChange={setSelectedTaskStatuses}
+              placeholder="Select a task status"
+              value={selectedTaskStatuses}
+            >
+              {taskStatuses.map((status) => (
+                <ComboboxOption
+                  key={status}
+                  displayName={taskStatusToCopy[status]}
+                  value={status}
+                />
+              ))}
+            </Combobox>
+            <Combobox
+              disabled={!hasResults}
+              label="Build Variant"
+              multiselect
+              onChange={setSelectedBuildVariants}
+              overflow="scroll-x"
+              placeholder="Select a build variant"
+              value={selectedBuildVariants}
+            >
+              {buildVariants.map((bv) => (
+                <ComboboxOption
+                  key={bv.buildVariant}
+                  displayName={bv.buildVariantDisplayName}
+                  value={bv.buildVariant}
+                />
+              ))}
+            </Combobox>
+          </FilterContainer>
+          {hasResults && (
+            <FilterSubheaderContainer>
+              <Body weight="medium">
+                {totalFilteredTestCount}/{totalTestCount} Failed{" "}
+                {pluralize("Test", totalFilteredTestCount)}
+              </Body>
+              <Button
+                disabled={hasFiltersApplied}
+                onClick={() => {
+                  setSelectedTaskStatuses([]);
+                  setSelectedBuildVariants([]);
+                  setTestName("");
+                }}
+                size="xsmall"
+              >
+                Clear Filters
+              </Button>
+            </FilterSubheaderContainer>
+          )}
+
+          <GroupedTestMapList groupedTestsMapEntries={groupedTestsMapEntries} />
+          {!hasMatchingResults && (
+            <BasicEmptyState
+              description="For additional analytics on tests please visit Honeycomb"
```

Comment:
Is there any partially prefilled query link we can provide here? I imagine an average user would have no idea where to begin on Honeycomb.

Reply:
Bump

Reply:
I think the goal behind this on the designs was to try to shift some patterns to honeycomb but taking a look there isn't much I could find for test specific results so I've omitted this description.